dist/minside.css

```diff
@@ -1166,6 +1167,15 @@ main{
   margin-top:3.5rem;
 }
 
+@media(min-width: 960px){
```

Comment:
Could we be using this here?

https://github.com/gramo-org/gramo-UI-kit/blob/8920dd600ff09174c28ce55c5ac24e56aab55af5/src/css/base/_variables.css#L77-L83

Reply:
we are. Thats the compiled css :P 

https://github.com/gramo-org/gramo-UI-kit/pull/40/files#diff-98cdca46f48aeed37ec3993b1ad51c6dR80

Reply:
Oh lol, that was the dist file.